--- arch_ps3.c	2012-07-30 09:54:35.373832800 +0800
+++ arch_ps3.c.new	2012-07-30 10:01:37.926001400 +0800
@@ -1104,15 +1104,18 @@
 void
 arch_preload_fonts(void)
 {
-  freetype_load_font("file:///dev_flash/data/font/SCE-PS3-VR-R-LATIN2.TTF",
+/* freetype_load_font("file:///dev_flash/data/font/SCE-PS3-VR-R-LATIN2.TTF",
 		     FONT_DOMAIN_FALLBACK, NULL);
-  freetype_load_font("file:///dev_flash/data/font/SCE-PS3-NR-R-JPN.TTF",
+ * freetype_load_font("file:///dev_flash/data/font/SCE-PS3-NR-R-JPN.TTF",
 		     FONT_DOMAIN_FALLBACK, NULL);
-  freetype_load_font("file:///dev_flash/data/font/SCE-PS3-YG-R-KOR.TTF",
+ * freetype_load_font("file:///dev_flash/data/font/SCE-PS3-YG-R-KOR.TTF",
 		     FONT_DOMAIN_FALLBACK, NULL);
-  freetype_load_font("file:///dev_flash/data/font/SCE-PS3-DH-R-CGB.TTF",
+ * freetype_load_font("file:///dev_flash/data/font/SCE-PS3-DH-R-CGB.TTF",
 		     FONT_DOMAIN_FALLBACK, NULL);
-  freetype_load_font("file:///dev_flash/data/font/SCE-PS3-CP-R-KANA.TTF",
+ * freetype_load_font("file:///dev_flash/data/font/SCE-PS3-CP-R-KANA.TTF",
+		     FONT_DOMAIN_FALLBACK, NULL);
+ */
+  freetype_load_font("file:///dev_hdd0/game/HTSS00003/USRDIR/FONT.TTF",
 		     FONT_DOMAIN_FALLBACK, NULL);
 }
 
